parlai/core/image_featurizers.py

```diff
@@ -142,6 +142,25 @@ def _image_mode_switcher(self):
 
         return switcher.get(self.image_mode)
 
+    @staticmethod
```

Comment:
perhaps a classmethod so it's easier to override? It's probably not necessary today.

Reply:
Good call. Added it. (I don't love that I added this function at all b/c the existing _image_features_switcher also has a list of some of the variants.)